src/scripts/Tabs.js

```diff
@@ -3,12 +3,29 @@ import ReactDOM from 'react-dom';
 import classnames from 'classnames';
 import { registerStyle } from './util';
 import DropdownButton from './DropdownButton';
+import { MenuItem } from './DropdownMenu';
 
 export default class Tabs extends Component {
-
   constructor(props) {
     super(props);
-    this.state = {};
+    const visibleTabs = [];
+    const hiddenTabs = [];
+
+    props.children.forEach((tab, index) => {
```

Comment:
It's recommended to use `React.Children.forEach` for iterating over react children in order to handle cases where children could not be an array